javascript/javascript3/week1/giphy.js

```diff
@@ -0,0 +1,17 @@
+const button = document.querySelector("button");
+button.addEventListener("click", () => {
+  url =
+    "https://api.giphy.com/v1/gifs/search?api_key=a7m3yo8CyGVk1GMcf3sLXZ13JyL783OJ&q=" +
+    document.getElementById("giphy-name").value +
```

Comment:
It better to check for the empty data first in separate variable and then assign it to the URL.

for example : let giphyName = document.getElementByID("giphy-name").value;

and then URL will be 

https://jsonformatter.curiousconcept.com/ + giphyName + "";